package.json

```diff
@@ -129,10 +130,9 @@
     }
   },
   "lint-staged": {
-    "**/*.{js,jsx,ts,tsx}": [
-      "npm run lint",
-      "npm run test:ci",
-      "git add ."
-    ]
+    "*.{js,jsx,ts,tsx}": [
+      "eslint"
+    ],
+    "*.js": "eslint --cache"
```

Comment:
Is it possible for this to be the same config as the components repository: 

```json
  "lint-staged": {
    "**/*.{js,jsx,ts,tsx}": [
      "npm run lint",
      "git add ."
    ]
  },
```

Reply:
I also think it should be applied to components library, not sure if that period in git add . is redundant and if that cached lint is worth it but I do believe we should add a pre commit test to the components project